class ConnectionException(Exception):
    pass


class UserException(Exception):
    pass
